cobalt/browser/h5vcc_runtime/public/mojom/h5vcc_runtime.mojom

```diff
@@ -14,9 +14,16 @@
 
 module h5vcc_runtime.mojom;
 
+interface DeepLinkListener {
+  ProcessDeepLink(string deep_link);
```

Comment:
Naming nit: I would call this `NotifyDeepLink()` or even better `NotifyOfWarnDeepLink()` because the caller of this method doesn't know what's going to happen with the deep link.